ui/shared/components/panel/variants/Predictions.jsx

```diff
@@ -38,38 +44,91 @@ const Prediction = ({ field, annotation, dangerThreshold, warningThreshold }) =>
+  { field: 'dann_score', name: 'DANN', warningThreshold: 0.93, dangerThreshold: 0.96 },
+  { field: 'revel_score', name: 'REVEL', warningThreshold: 0.5, dangerThreshold: 0.75 },
+  { field: 'eigen_phred', name: 'EIGEN', warningThreshold: 1, dangerThreshold: 2 },
+  { field: 'mpc_score', name: 'MPC', warningThreshold: 1, dangerThreshold: 2 },
+  { field: 'primate_ai_score', name: 'PMT AI', warningThreshold: 0.5, dangerThreshold: 0.7 },
+  { field: 'polyphen' },
+  { field: 'sift' },
+  { field: 'mut_taster' },
+  { field: 'fathmm' },
+  { field: 'metasvm' },
+  { field: 'gerp_rs' },
+  { field: 'phastcons100vert' },
+]
 
-const Predictions = ({ annotation }) => {
-  return annotation ?
-    <div>
-      <Prediction field="polyphen" annotation={annotation} />
-      <Prediction field="sift" annotation={annotation} />
-      <Prediction field="mut_taster" annotation={annotation} />
-      <Prediction field="fathmm" annotation={annotation} />
-      <Prediction field="cadd_phred" annotation={annotation} dangerThreshold={20} warningThreshold={10} />
-      <Prediction field="dann_score" annotation={annotation} dangerThreshold={0.96} warningThreshold={0.93} />
-      <Prediction field="revel_score" annotation={annotation} dangerThreshold={0.75} warningThreshold={0.5} />
-      <Prediction field="eigen_phred" annotation={annotation} dangerThreshold={1} warningThreshold={2} />
-      <Prediction field="mpc_score" annotation={annotation} dangerThreshold={2} warningThreshold={1} />
-    </div>
-    : null
-}
+export default class Predictions extends React.Component {
+  static propTypes = {
+    annotation: PropTypes.object,
+  }
 
-Predictions.propTypes = {
-  annotation: PropTypes.object,
+  constructor(props) {
+    super(props)
+
+    this.state = { showMore: false }
+  }
+
+  render() {
+    const { annotation } = this.props
+
+    if (!annotation) {
+      return null
+    }
+
+    const predictorFields = PREDICTOR_FIELDS.filter(predictorField => annotation[predictorField.field] != null)
+    const predictorFieldsAboveTheFold = predictorFields.slice(0, NUM_TO_SHOW_ABOVE_THE_FOLD)
+    const morePredictorFields = predictorFields.slice(NUM_TO_SHOW_ABOVE_THE_FOLD)
+    return (
+      <div>
+        {
+          predictorFieldsAboveTheFold.map(predictorField =>
+            <Prediction key={predictorField.field} annotation={annotation} {...predictorField} />)
+        }
+        {
+          morePredictorFields ?
+            <Transition.Group animation="fade down" duration="500">
+              {
+                this.state.showMore ?
+                  <div>
+                    {
+                      morePredictorFields.map(predictorField => <Prediction key={predictorField.field} field={predictorField.field} annotation={annotation} />)
+                    }
+                    <StyledButtonLink onClick={() => this.setState({ showMore: false })}>hide</StyledButtonLink>
```

Comment:
rather than define functions within props which can cause unnecessary re-renders, could we instead have a `toggleShowMore` function defined on the class that toggles the state